### PR TITLE
PS-8889 Add quickstart (8.1)

### PR DESCRIPTION
**Pull request by patrickbirch**
new file:   docs/quickstart-database.md
	new file:   docs/quickstart-docker.md
	new file:   docs/quickstart-exit.md
	new file:   docs/quickstart-next-steps.md
	modified:   docs/quickstart-overview.md
	new file:   docs/quickstart-queries.md

### CODE REVIEW

File: docs/docker-exit.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+The default length of time before stopping a container is 10 seconds. A large instance cannot dump the data from memory to disk within that time. With this type of instance, add the `--time` or the `-t` option to docker stop:
+
+```{.bash data-prompt="$"}
+$ docker stop ps -t 600
+```
+
+## Exit the container
+
+If you are in the interactive shell, use `CTRL-D` or exit to exit the session.
```
Comment (alina-derkach-oaza) [inline docs/docker-exit.md]:
```suggestion
If you are in the interactive shell, use `CTRL-D` or `exit` to exit the session.
```

File: docs/docker-exit.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+If you have a non-shell process running, interrupt the process with `CTRL-C` before using `CTRL-D` or `exit.`
+
+
+## Removing the container
```
Comment (alina-derkach-oaza) [inline docs/docker-exit.md]:
```suggestion
## Remove the container
```

File: docs/docker-storage.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+The `-v /local/datadir:/var/lib/mysql` option mounts the `/local/datadir` directory on the host to `/var/lib/mysql` in the container,
+which is the default data directory used by Percona Server for MySQL.
+
+Do not add MYSQL_ROOT_PASSWORD to the `docker run` command if the data directory contains subdirectories, files, or data.
+
+!!! note 
+
+    If you have SELinux enabled, assign the relevant policy type to the new data directory so that the container will be allowed to access it:
+
+```
```
Comment (alina-derkach-oaza) [inline docs/docker-storage.md]:
```suggestion
    ```
```

File: docs/docker-storage.md [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+[root@docker-host] $ chcon -Rt svirt_sandbox_file_t /local/datadir
```
Comment (alina-derkach-oaza) [inline docs/docker-storage.md]:
```suggestion
    [root@docker-host] $ chcon -Rt svirt_sandbox_file_t /local/datadir
```

File: docs/docker-storage.md [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+```
```
Comment (alina-derkach-oaza) [inline docs/docker-storage.md]:
```suggestion
    ```
```

File: docs/docker-logs.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
+# Docker troubleshooting
```
Comment (alina-derkach-oaza) [inline docs/docker-logs.md]: you have two docks with the same name. This one and docs/docker-troubleshooting.md

Comment (alina-derkach-oaza) [reply docs/docker-logs.md]: And the docs are the same

File: mkdocs-base.yml [hunk 7]
```diff
@@ -221,6 +226,10 @@ nav:
     - Docker:
       - Install using Docker: docker.md
       - docker-config.md
+      - docker-logs.md
+      - docker-troubleshooting.md
```
Comment (alina-derkach-oaza) [inline mkdocs-base.yml:229]: Remove either  docker-logs.md or docker-troubleshooting.md

File: docs/docker-logs.md (hunk 6 above)
Comment (patrickbirch) [reply docs/docker-logs.md]: removed

**Pull request closed without merging**